tests/Maker/AdminMakerTest.php

```diff
@@ -109,7 +108,7 @@ public function testExecute(): void
         $this->output = new StreamOutput(fopen('php://memory', 'w', false));
 
         $this->io = new ConsoleStyle($this->input, $this->output);
-        $fileManager = new FileManager(new Filesystem(), '.');
+        $fileManager = new FileManager(new Filesystem(), new AutoloaderUtil(new ComposerAutoloaderFinder('Sonata\AdminBundle\Tests')), $this->projectDirectory.'/config/'.$this->servicesFile);
```

Comment:
@franmomu since this particular change, I have a dirty working directory after running the tests:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	tests/Admin/FooAdmin.php
	tests/Controller/FooAdminController.php

```

Shouldn't these be generated in `/tmp`?

Reply:
let me take a look